src/test/java/DefaultRecheckIgnoreTest.java

```diff
@@ -0,0 +1,30 @@
+import static de.retest.recheck.ignore.FilterLoader.load;
+import static java.nio.file.Paths.get;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.LoggerFactory;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+import de.retest.recheck.review.ignore.io.ErrorHandlingLoader;
+
+class DefaultRecheckIgnoreTest {
+
+	@Test
+	void default_recheck_ignore_should_load_without_exception() throws Exception {
+		final ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
```

Comment:
We have an [extension](https://github.com/retest/recheck/blob/bd0f8ded6cb6cb4b98dcc466e5b622170a4db660/src/test/java/de/retest/recheck/review/workers/LoadAndSaveFilterWorkerTest.java#L29) that does this.

Reply:
Wow, didn't know! Thanks